segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageTests.java

```diff
@@ -68,6 +69,11 @@ public void after() throws Exception {
         super.before();
     }
 
+    @Override
+    protected int getThreadPoolSize() {
+        return THREAD_POOL_SIZE;
```

Comment:
Why do you need 10 threads for this? 1-3 is more than sufficient.

GH Actions doesn't give you more than 2 processors, and you may barely be able to use one of them since we run our tests in parallel. I don't understand the benefit that running your tests with 10 threads may bring.

Reply:
Same everywhere else below.

Reply:
My intention here is to add more randomness in task scheduling. With more threads the probability of getting same scheduling every time decreases while increasing the probability of hitting corner case related to timing issues.

Reply:
Fixed. Using 3 threads.